assets/i18n/en.json

```diff
@@ -562,26 +565,25 @@
-    "I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_5": "<strong>Represent it accurately</strong> - The learning objective is shown to learners who are browsing explorations, and should reflect the content of your exploration. The exploration should deliver all the content promised in the learning objective and in the exploration itself.",
-    "I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_6": "<strong>Stay interesting!</strong> - The best explorations tell a story, and have useful feedback. They give learners a chance to reason about concepts, try out new knowledge, and receive useful feedback on their work.",
-    "I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_PARAGRAPH_1": "When an exploration is published, it is credited to you, and becomes available for anyone to learn from. In order to create a good learning experience for students, here's a few tips we've found useful:",
-    "I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_PARAGRAPH_2": "For additional help on crafting useful feedback and good explorations, check out our <a href=\"http://oppia.github.io/#/DesignTips\" class=\"inline-links\">Design Tips</a> page.",
-    "I18N_TEACH_PAGE_PUBLICATION_POLICY_HEADING": "Publication policy",
-    "I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_1": "Any owner of a private exploration can publish it at any time. However, low-quality explorations may be unpublished by the moderators, with feedback on how to improve them.",
-    "I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_2": "Explorations are meant to be continuously improved, and explorations that haven't been edited for a long time may be considered \"orphaned\". In this case, ownership may be handed to the Oppia community at large (at moderator discretion), so that the exploration can continue to be improved.",
-    "I18N_TEACH_PAGE_TAB_PARTICIPATION_PLAYBOOK": "Participation Playbook",
-    "I18N_TEACH_PAGE_TAB_TEACH_WITH_OPPIA": "Teach with Oppia",
+    "I18N_TEACH_BENEFITS_TITLE": "Our Benefits",
+    "I18N_TEACH_PAGE_ACTION_START_LEARNING": "Start Learning",
+    "I18N_TEACH_PAGE_CLASSROOM_BUTTON": "VISIT CLASSROOM",
+    "I18N_TEACH_PAGE_CLASSROOM_CONTENT": "In the Classroom, you can find a set of lessons that the Oppia team has designed and tested to make sure they’re effective and fun for all learners. All of the lessons have been reviewed by teachers and experts, so you can feel assured that your students are getting an effective education, while learning at their own pace.",
+    "I18N_TEACH_PAGE_CLASSROOM_TITLE": "Learn from Oppia’s Tested and Verified Lessons",
+    "I18N_TEACH_PAGE_CONTENT" : "Oppia is an engaging, new approach to online learning that specifically is made to ensure that everyone has access to quality education.",
+    "I18N_TEACH_PAGE_HEADING": "Oppia for Parents, Teachers and Gaurdians",
```

Comment:
"Gaurdians" is misspelled. Should be "Guardians"

Reply:
oops sorry. thank you

Reply:
Has this been updated in the videos? It still seems like this is misspelled there

Reply:
sorry, I didn't update the videos.